packages/stable/src/api/files/filesApi.ts

```diff
@@ -233,6 +248,52 @@ export class FilesAPI extends BaseResourceAPI<FileInfo> {
     );
     return this.addToMapAndReturn(response.data.items, response);
   }
+  /**
+   * [Init a multipart file upload](https://api-docs.cognite.com/20230101/tag/Files/operation/initMultiPartUpload)
+   *
+   * ```js
+   * const numberOfParts = 5;
```

Comment:
Probably worth noting in the example here that the number of parts needs to be carefully calculated, to adhere to the part size requirements in the API documentation, which states:

> Each file part must be larger than 5 MiB, and smaller than 4000 MiB. The file part for the last uploadURL can be smaller than 5 MiB.

Reply:
added that comment